game/tst/TestModuleTest.py

```diff
@@ -0,0 +1,9 @@
+import unittest
+
+import TestModule
+
+
+class TestModuleTest(unittest.TestCase):
+
+    def test_getRandom(self):
+        self.assertEqual(4, TestModule.get_random_number())
```

Comment:
This is called a unit test. You can use these to define the contracts for your functions. In this case we're asserting that every time we call get_random_number)_ it will return a result of 4.

Reply:
Contracts for your function? I'm not familiar with that terminology, I don't think

Reply:
A contract is simple what the function will do. "Given input X I will do Y".
In this case "I will always return 4".